parameter_generation/params_sent_weights.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+'''
+    Executable to generate parameters for sentence
+    disambiguation weights, namely the files:
+    - storage/data_nn_disambiguator/X_train.npy
+    - storage/data_nn_disambiguator/X_test.npy
+    - storage/data_nn_disambiguator/y_train.npy
+    - storage/data_nn_disambiguator/y_test.npy
+    - storage/trained_weights.npy
+'''
+
+import os
+import sys
+project_path = os.path.join(os.path.dirname(__file__), '../')
+sys.path.append(project_path)
```

Comment:
I'm thinking that's probably the way to go; I think we should probably end up separating out everything into its own sub-module. Right now it really doesn't matter since we only have sentence separation, but we'll eventually probably want one sub-module per step in the pipeline